vc/pokeyellow.patch.template

```diff
@@ -0,0 +1,533 @@
+Mode = 1
+Address = {HEX @+1}
+Fixcode = {PATCH +1}
+
+[MiniGame-HighScorePC@MiniGameHighScorePC]
+Mode = 1
+Address = {HEX @+1}
+Fixcode = {PATCH +1}
+
+[send_send_buf2]
+Mode = 2
+Address = {HEX @}
+Type = 29
+
+[send_send_buf2_ret]
+Mode = 2
+Address = {HEX @}
+Type = 30
+
+[send_byt2]
+Mode = 2
+Address = {HEX @+5}
+Type = 31
+
+[send_byt2_ret]
+Mode = 2
+Address = {HEX @}
+Type = 32
+
+[send_dummy]
+Mode = 2
+Address = {HEX @}
+Type = 33
+
+[send_dummy_end]
+Mode = 2
+Address = {HEX @}
+Type = 34
+
+[FIGHT]
+Mode = 1
+Address = {HEX @+1}
+Fixcode = {PATCH +1}
+
+[FIGHT2]
+Mode = 1
+Address = {HEX @+1}
+Fixcode = {PATCH +1}
+
+[Network10]
+Mode = 1
+Address = {HEX @+1}
+Fixcode = {PATCH +1}
+
+[Network11]
+Mode = 1
+Address = {HEX @+1}
+Fixcode = {PATCH +1}
+
+[Network424]
+Mode = 2
+Address = {HEX @}
+Type = 4
+
+[Network200]
+Mode = 2
+Address = {HEX @}
+Type = 4
+
+;[RND_HACK]
+;Mode = 1
+;Address = 0x3E70
+;Fixcode = a10: 3E 3D 00 00 00 00 00 00 00 00 
+
+;[PLAY_HACK]
+;Mode = 2
+;Address = 0x2238
+;Type = 5
+
+
+;[PLAY_MUTE]
+;Mode = 1
+;Address = 0x150
+;Fixcode = 0xC9
+
+[PLAY_MUTE]
+Mode = 1
+Address = {HEX @+1}
+Fixcode = {PATCH +1}
+
+[PLAY_HACK]
+Mode = 2
+Address = {HEX @}
+Type = 6
+
+[PLAY_NEW]
+Mode = 2
+Address = {HEX @}
+Type = 21
+
+[Network_RECHECK]
+Mode = 2
+Address = {HEX @}
+Type = 7
+
+[Network_STOP]
+Mode = 2
+Address = {HEX @}
+Type = 8
+
+[Network_END]
+Mode = 2
+Address = {HEX @}
+Type = 9
+
+[Network_RESET]
+Mode = 2
+Address = {HEX @}
+Type = 10
+
+[linkCable fake begin]
+Mode = 2
+Address = {HEX @}
+Type = 16
+
+[linkCable fake end]
+Mode = 2
+Address = {HEX @}
+Type = 17  
+  
+[linkCable block input]
+Mode = 2
+Address = {HEX @+5}
+Type = 18
+
+;[save game start]
+;Mode = 2
+;Address = 0x5A84
+;Type = 19
+
+[save game end]
+Mode = 2
+Address = {HEX @}
+Type = 20
+
+[Change_MSG]
+Mode = 1
+Address = {HEX @+1}
+Fixcode = {PATCH_ +1 20}
+
+[fight_ret]
+Mode = 1
+Address = {hex @}
+Fixcode = {PATCH}
+
+[fight_ret_c]
+Mode = 2
+Address = {hex @}
+Type = 98
+
+;Mem Write: pc32 = 0xf992d addr = 0xc634 value = 0xe
+;Mem Write: pc32 = 0xf9e51 addr = 0xc635 value = 0x1
+[FPA 031801 Begin1]       
+Mode = 3                   
+Type = 0                   
+Address = {hex @}         
+MotionBEnable0 = 3         
+MotionBlur0 = 28     
+ConditionType = 0                    
+ConditionValueA = {dws_ wYellowIntroCurrentScene}    
+ConditionValueB = {dws_ ==                      }      
+ConditionValueC = {dws_ YellowIntroScene14_Index}      
+
+;34 6F 26 00
+;Mem Write: pc32 = 0xf9e51 addr = 0xc635 value = 0x33   
+[FPA 031801 End1]                
+Mode = 3                         
+Type = 1                         
+Address = {hex @}                
+ConditionType = 0                
+ConditionValueA = {dws_ wYellowIntroCurrentScene wYellowIntroSceneTimer}     
+ConditionValueB = {dws_ ==                       ==                    }     
+ConditionValueC = {dws_ YellowIntroScene14_Index 0x33                  }     
```

Comment:
Is this an index into something, or just some arbitrary timer value?

Reply:
From what I understand this is an arbitrary timer value.. From what I can tell; the scenes use this value sometimes to determine when to transition to other sequences. 0x33 doesn't like up with anything particular that I am able to tell.